index.js

```diff
@@ -191,20 +142,30 @@ var Umzug = module.exports = redefine.Class({
 
     if (typeof options === 'undefined') {
       return getExecuted().bind(this).then(function (migrations) {
-        return migrations[0] ? this.down(migrations[0].file) : Bluebird.resolve([]);
+        return migrations[0]
+          ? this.down(migrations[0].file)
+          : Bluebird.resolve([]);
       });
-    } else if (typeof options === 'string') {
-      return this.down([ options ]);
+    } else {
+      return this._run(false, options, getExecuted.bind(this));
+    }
+  },
+
+  _run: function(up, options, rest) {
```

Comment:
Can we call the first arg `method` or `direction`. I dislike boolean arguments quite a lot :)